packages/api-db-mongodb/src/db/peer.ts

```diff
@@ -28,6 +28,7 @@ export class MongoDBPeerAdapter implements DBPeerAdapter {
       return {
         name: '',
         themeColor: '#000000',
+        themeFontColor: '#d67c15',
```

Comment:
Do you want this color default?

Reply:
Maybe not the nicest color indeed, I'll change it to white